components/motor/gpio/motor_encoder.go

```diff
@@ -22,31 +22,6 @@ import (
 	rdkutils "go.viam.com/rdk/utils"
 )
 
```

Comment:
In another ticket - let's add a global variable `rpmDebug` and add back the debug logs
```suggestion
if rpmDebug {
m.logger.Debug(....)
}
```

Reply:
added a ticket to the epic